demo/src/data/example_pane_notation.rs

```diff
@@ -0,0 +1,50 @@
+use pretty::{Color, CursorVis, DocLabel, DocPosSpec, PaneNotation, PaneSize, Style};
+
+pub fn make_example_pane_notation() -> PaneNotation {
+    let active = PaneNotation::Doc {
+        label: DocLabel::ActiveDoc,
+        cursor_visibility: CursorVis::Show,
+        scroll_strategy: DocPosSpec::CursorHeight { fraction: 0.6 },
+    };
+
+    let key_hints_name = PaneNotation::Doc {
+        label: DocLabel::KeymapName,
+        cursor_visibility: CursorVis::Hide,
+        scroll_strategy: DocPosSpec::Beginning,
+    };
+
+    let key_hints = PaneNotation::Doc {
+        label: DocLabel::KeyHints,
+        cursor_visibility: CursorVis::Hide,
+        scroll_strategy: DocPosSpec::Beginning,
+    };
+
+    let messages = PaneNotation::Doc {
+        label: DocLabel::Messages,
+        cursor_visibility: CursorVis::Hide,
+        scroll_strategy: DocPosSpec::Beginning,
+    };
+
+    let divider = PaneNotation::Fill {
+        ch: '=',
```

Comment:
Characters don't work well anyways because there's no `DynWidth` option for the text in the status bar. Maybe we should reverse the background instead.

Reply:
And PaneNotations don't really support applying a Style to a whole Pane right now, so I'm leaving this as-is until those `pretty` changes shake out.